sklearn/preprocessing/tests/test_data.py

```diff
@@ -474,6 +473,13 @@ def test_standard_scaler_partial_fit_numerical_stability():
         assert_allclose(scaler_incr.var_, scaler.var_, rtol=tol)
         assert_allclose(scaler_incr.scale_, scaler.scale_, rtol=tol)
 
+    X = np.ones(10).reshape(5, 2)
+    X[0][0] = np.nan
+    X_csr = sparse.csr_matrix(X)
+    scaler = StandardScaler(with_mean=False)
+    scaler.partial_fit(X_csr)
+    assert_almost_equal(scaler.mean_, 1.0)
+
```

Comment:
I would try to have a test with only nan value in X or at least a complete column